pkg/operator/hive/hive.go

```diff
@@ -5,6 +5,8 @@ import (
 	"context"
 	"crypto/md5"
 	"fmt"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
These imports should be in a section between the section with logrus and the section with hive.